server/dummyData/dummySurveys.js

```diff
@@ -16,35 +15,25 @@ module.exports = [
     anonymous: false,
     recipients: [
       {
-        employeeId: ObjectID('507f1f77bcf86cd799439014'),
         completed: false,
       },
```

Comment:
why have you deleted the Ids? They're pretty important!

Reply:
Fixed.